### PR TITLE
D9 automated Tests

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Updated circleci to test D8 and D9

# Need Review By (Date)
- 1/15/2021

# Urgency
- medium

# Steps to Test
1. review passing tests

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -96,5 +96,5 @@ workflows:
   tests:
     jobs:
       - run-coverage
-#      - run-codeception
+      - run-codeception
```
Comment (sherakama) [inline .circleci/config.yml:99]: 😍

Comment (pookmish) [reply .circleci/config.yml:99]: Don't get your hopes up. haha. This is just to test an update to caravan for D9 testing.

File: .circleci/config.yml [hunk 2]
```diff
@@ -82,19 +98,19 @@ workflows:
...
           filters:
             branches:
               only:
-                - 8.x-2.x
-  tests:
-    jobs:
-      - run-coverage
-#      - run-codeception
-
+                - 8.x-1.x
```
Comment (sherakama) [inline .circleci/config.yml:116]:
```suggestion
                - 8.x-2.x
```

**Review by sherakama — APPROVED** [review]
OK

**Pull request merged**